portal-impl/src/custom-sql/portal.xml

```diff
@@ -1056,6 +1056,29 @@
 				[$TYPE$]
 		]]>
 	</sql>
+	<sql id="com.liferay.portal.kernel.service.persistence.RoleFinder.countU_GByRoleId">
+		<![CDATA[
+			SELECT SUM(TEMP_TABLE) AS COUNT_VALUE
```

Comment:
That makes sense.  I was surprised we didn't do this anywhere else, but since I couldn't find any existing examples I went with what worked.  

@brianchandotcom do you have an opinion on this?